### PR TITLE
Delete user account

### PR DESCRIPTION
**Pull request by arb1ona**
I have pushed the code with the intent to delete the user account and its articles associated with it. 
1. I can delete the user eventually but after deletion it doesn't show the notification

### CODE REVIEW

**Comment by arb1ona** [comment]
`flash.now` made the work :)
Its working now

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -40,6 +40,14 @@ def create
         end
     end
 
+    def destroy
+      @user.destroy
+      session[:user_id] = nil #otherwise app willl through an error
+      flash.now[:alert] = "Account and all associated articles successfully deleted"
+      render 'new'
+      # redirect_to articles_path
```
Comment (KarolinaLaski) [inline app/controllers/users_controller.rb:48]: I think you could also use `redirect_to articles_path, notice: "Account and all associated articles successfully deleted"` 🤔

**Comment by arb1ona** [comment]
I tried that way, just as the tutorial. But keeps redirecting me at Login
page with notification of 'You must be logged in to perform this action',
while on the same time the user has been deleted.🤔

**Comment by KarolinaLaski** [comment]
> I tried that way, just as the tutorial. But keeps redirecting me at Login page with notification of 'You must be logged in to perform this action', while on the same time the user has been deleted.🤔

Oh yes, that is correct. The user and session was deleted and you need to be logged in in order to see the articles, right? Unless it doesn't then its a little weird it shows that. Do you require user to be logged in to see all the articles?

**Comment by arb1ona** [comment]
Yes, I require users to login to see the articles. But the notification
that the user has been deleted isn't showed up - using •
flash[:notice]="Account is deleted"
• redirect_to articles_path

**Pull request merged**